### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_ColorContrast : The Color contrast of focus on "ViewMessage" button is less than 3:1

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3461]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.11;#Win32;#A11ySev2;#Benchmark;#ColorContrast;#FTP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Navigate to View message button
7. Open AI4D and Check the color contrast of focus on "ViewMessage" button.

### **Actual:**
The color contrast ratio of focus on "ViewMessage" button is less than 3:1

**Similar issue also observed for following scenarios:**

1. Queues-->AddMessage
2. Queues-->DequeueMessage
3. Queues-->ClearQueue

### **Expected:**
The color contrast ratio of focus on "ViewMessage" button should be greater than 3:1.

### **User Impact:**
Low vision users may not able to see the control has keyboard focus.

### **Attachment**
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prraha_microsoft_com/EZckjtgwyF9CoBdsfFjXyxEBhv9eUXHIVyFz963VDQQB1A?e=O9IFAR

**Comment by MRayermannMSFT** [comment]
See screenshot from bug #3457

**Comment by MRayermannMSFT** [comment]
Chitrarupa said this was fixed in email from 10/19.

**Comment by pranshu512** [comment]
![image](https://user-images.githubusercontent.com/64009219/96561676-81d19300-12dd-11eb-8247-4f5dd09501db.png)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
